src/Components/Classes/Fprm/newclass.module.css

```diff
@@ -0,0 +1,104 @@
+.addClass {
+  width: 70vh;
+  height: 10vh;
+  padding-top: 30px;
+}
+.updateClass {
+  width: 70vw;
+  height: 46vh;
```

Comment:
We could avoid defining a height in this case, since in some resolutions the whole form is not fully displayed.

Reply:
fixed